java/maven.embedder/nbproject/project.xml

```diff
@@ -182,6 +182,7 @@
                 <friend>io.github.jeddict.runtime</friend>
                 <friend>io.github.jeddict.rest.generator</friend>
                 <!-- XXX <subpackages> not permitted by schema -->
+                <friend>org.netbeans.modules.micronaut</friend>
```

Comment:
I don't like this. `micronaut` module shall not have direct dependency on Maven (embedder).

Reply:
would be gradle+maven micronaut bridge modules more approppriate ?